articles/appliance/geo-ha/index.md

```diff
@@ -27,6 +27,16 @@ The arbiter does not store data or execute application logic, but acts as a witn
 
 > Ports 27017 and 7777 must be open between all instances in the cluster.
 
+### Arbiter
+
+The Arbiter node acts as an independent witness to the primary and secondary data centers. Since it isn’t storing data and doesn’t run any services, it can be a small instance with two cores and 4GB of memory.
+
+### Geographically-Aware Global Load Balancer/DNS Failover Configuration
+
+You will need to deploy a global load balancer that supports an active/standby configuration. This will be configured to fail over to the secondary site if the primary site load balancer is unavailable.
```

Comment:
"to fail over to the secondary site" does not read that well (in my opinion). Maybe a rephrase would be good here? "This will be configured to fall back on the secondary site, .. " or something?